spec/defines/topic_spec.rb

```diff
@@ -18,12 +18,33 @@
         {
           'ensure'             => 'present',
           'zookeeper'          => 'localhost:2181',
-          'replication_factor' => '1',
```

Comment:
hrm do we need to assume that people used quoted integers? If so, the new datatype would be a breaking change.

Reply:
How about a Variant[String, Integer]) and throw a deprecation warning on string-type?

Reply:
sounds good!